### DISCUSSION TITLE
Duplicate deb file name for arm

### DISCUSSION THREAD

**Discussion by yoshioki** [authelia/authelia#7745] [General]
Hi.

There are two arm64 deb files posted in [releases](https://github.com/authelia/authelia/releases).

I think one of them is the musl version.

They cannot be distinguished because the file names are the same.

Since the URLs are the same, I don't know which one will be downloaded.

Could you please change the file name ?

![Releases-·-authelia-authelia](https://github.com/user-attachments/assets/712594a6-b9cf-4cb6-8118-a16cf58b5996)

**Comment by yoshioki** [comment]
sorry

It was the difference between arm and amd.

Is .deb for arm the musl version?

**Reply by nightah** [reply]
Debian doesn't support musl out of the box.

They're built with Debian so it's glibc based.
